modules/core-updates/9.0.md

```diff
@@ -85,6 +85,40 @@ Some dependencies are still present but were upgraded which comes with their own
 - [Services translator, security.csrf.token_manager, session, session.flash_bag no longer public](https://github.com/PrestaShop/PrestaShop/pull/35069)
 - [Replace AdminSecurity annotation with attributes](https://github.com/PrestaShop/PrestaShop/pull/35212)
 
+### BO login and authorization migrated to Symfony
+
+The login page to the back-office has been migrated to Symfony, along ith it the system of authorization in the BackOffice is now also based on Symfony, it implies several things:
+- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side and very few data are kept on the browser side, for retro-compatibility we still populate the legacy cookie so that you can **read** it but modifying/writing it will likely no longer have the effect you would expect (if any)
```

Comment:
```suggestion
- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side, and very few data are kept on the browser side, for retro-compatibility we still populate the legacy cookie so that you can **read** it, but modifying/writing it will likely no longer have the effect you would expect (if any)
```

as @Hlavtox mentioned, it requires some clarification because it is probably not true for most of the scenarios cookies have been used by community developers

Reply:
Yeah I modified this, let me know after I push my changes it it's clearer